src/components/InputItem/InputItem.js

```diff
@@ -14,23 +14,31 @@ class InputItem extends React.Component {
   };
 
   onButtonClick = () => {
+    let error = false;
     this.setState({
       inputValue: ''
     });
-    if (this.state.inputValue !== '') {
-      this.props.onClickAdd(this.state.inputValue);
+    if (this.state.inputValue == '') {
       this.setState({
-        label: "Добавить задание",
-        id: "standard-basic",
-        error: false
-      })
-    } else {
+        error: true,
+        id: "standard-error-helper-text",
+        label: "Введите задание!",
+      });
+    } else if (this.props.items.filter(item => item.value == this.state.inputValue).length !== 0) {        
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/WebHeroSchool/agatta17_todo/pullRequest?prid=6081322)